AfricanReligions/localisation/afr_general_localization.csv

```diff
@@ -251,6 +251,12 @@ heir_of_dihya;Heir of Dihya;;;;;;;;;;;;;x
 heir_of_dihya_desc;This character has fought for the Amazigh faith as Queen Dihya did in the days of yore.;;;;;;;;;;;;;x
 suffete;Suffete;;;;;;;;;;;;;x
 suffete_desc;This character is the highest magistrate of the Carthaginian Empire.;;;;;;;;;;;;;x
+follower_of_serer;Serer;;;;;;;;;;;;;x
+follower_of_serer_desc;The Serer mastery of spiritual knowledge is greater than that of any other and depends on the interaction between the three worlds: the invisible world, the terrestrial world and the nocturnal world.;;;;;;;;;;;;;x
```

Comment:
For this description and the following, there are localizations on the "General Questions" page of the spreadsheet that you should use.